src/components/MapPoiTooltip/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import { Popup } from 'react-map-gl'
+
+export interface MapPoiTooltipType {
+  title?: string
+  category?: string
+  info?: string
+  coordinates: {
+    latitude: number
+    longitude: number
+  }
+}
+
+export const MapPoiTooltip: FC<MapPoiTooltipType> = ({
+  title,
+  category,
+  info,
+  coordinates,
+}) => {
+  return (
+    <Popup
+      latitude={coordinates.latitude}
+      longitude={coordinates.longitude}
+      closeButton={false}
+      className="w-64 rounded-2xl"
+    >
+      {title && <h4 className="text-xl leading-6 text-gray-900">{title}</h4>}
```

Comment:
These conditions here imply that there is the possibility that a tooltip without content is shown. I would add a condition that prevents the tooltip to show at all or that indicates that no data is available if neither `title`, `category`, nor `info` are available.

Reply:
True, these conditions actually don't make sense. I've made a change now that requires `title` and `category` in `MapPoiTooltip` as they are crucial for understanding the POI. Without these provided, the component won't be rendered at all.